d2target/d2target.go

```diff
@@ -17,6 +17,11 @@ const (
 	MAX_ICON_SIZE     = 64
 )
 
+type DiagramObject interface {
```

Comment:
unlike in Java, Go interfaces belong at site of usage, not definition. (so multiple usages can have their own interface)

Reply:
cool, TIL..
Now it somehow makes sense why the interface is not set with the struct..